File: clients/javascript/src/models/serverState.ts
// this file is @generated

export enum ServerState {
    Leader = 'Leader',
    Follower = 'Follower',
    Learner = 'Learner',
    Candidate = 'Candidate',
    Shutdown = 'Shutdown',
    Unknown = 'Unknown',
    }

export const ServerStateSerializer = {
    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _fromJsonObject(object: any): ServerState {
        return object;
    },

    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _toJsonObject(self: ServerState): any {
        return self;
    }
}

File: z-clients/python/diom/models/server_state.py
# this file is @generated
from enum import Enum


class ServerState(str, Enum):
    LEADER = "leader"
    FOLLOWER = "follower"
    LEARNER = "learner"
    CANDIDATE = "candidate"
    SHUTDOWN = "shutdown"
    UNKNOWN = "unknown"

    def __str__(self) -> str:
        return str(self.value)
